DependencyInjection/Configuration.php

```diff
@@ -51,6 +56,9 @@ public function getConfigTreeBuilder()
             ->arrayNode('mail')
                 ->addDefaultsIfNotSet()
                 ->children()
+                    ->scalarNode('helper')
+                        ->defaultValue(self::SWIFT_MAILER_HELPER)
+                        ->end()
```

Comment:
end should be on the same level as the opening node.

```suggestion
                    ->end()
```